src/Command/SF1500/SF1500LookupCommand.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+/**
+ * This file is part of itk-dev/serviceplatformen.
+ *
+ * (c) 2020 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace ItkDev\Serviceplatformen\Command\SF1500;
+
+use GuzzleHttp\Client;
+use Http\Adapter\Guzzle6\Client as GuzzleAdapter;
+use Http\Factory\Guzzle\RequestFactory;
+use ItkDev\AzureKeyVault\Authorisation\VaultToken;
+use ItkDev\AzureKeyVault\KeyVault\VaultSecret;
+use ItkDev\Serviceplatformen\Certificate\AzureKeyVaultCertificateLocator;
+use ItkDev\Serviceplatformen\Certificate\CertificateLocatorInterface;
+use ItkDev\Serviceplatformen\Certificate\FilesystemCertificateLocator;
+use ItkDev\Serviceplatformen\Service\SF1500\SF1500;
+use ItkDev\Serviceplatformen\Service\SF1500\SF1500XMLBuilder;
+use ItkDev\Serviceplatformen\Service\SF1514\SF1514;
+use ItkDev\Serviceplatformen\Service\SoapClient;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidOptionException;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\PropertyAccess\PropertyAccessor;
+
+class SF1500LookupCommand extends Command
+{
+    const SF1500_TEST_MANAGER_FUNKTIONSTYPE_ID = '46c73630-f7ad-4000-9624-c06131cde671';
+    const SF1500_PROD_MANAGER_FUNKTIONSTYPE_ID = 'to_be_found';
```

Comment:
I think this should be an option. We don't want to change the code if the ids change.

Reply:
Changed into option.